File: BOJ/2309_1.py
'''
    일곱 난쟁이
    https://www.acmicpc.net/problem/2309
'''
import sys
dwarf = []
total = 0
for _ in range(9):
    height = int(sys.stdin.readline())
    total += height
    dwarf.append(height)

dwarf.sort()

for i in range(9):
    for j in range(i + 1, 9):
        if total - dwarf[i] - dwarf[j] == 100:
            for k in range(9):
                if k != i and k != j:
                    print(dwarf[k])
            sys.exit(0)

File: BOJ/2309_1.cpp
// 일곱 난쟁이
// https://www.acmicpc.net/problem/2309
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;
vector<int> dwaf(9);
int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);

    int total = 0;
    for (int i = 0; i < 9; i++)
    {
        cin >> dwaf[i];
        total += dwaf[i];
    }

    sort(dwaf.begin(), dwaf.end());

    for (int i = 0; i < 9; i++)
    {
        for (int j = i + 1; j < 9; j++)
        {
            int now = total - dwaf[i] - dwaf[j];
            if (now == 100)
            {
                for (int k = 0; k < 9; k++)
                    if (k != i && k != j)
                        cout << dwaf[k] << '\n';
                return 0;
            }
        }
    }
}